framework/set/resources/rancher2/setProvidersAndUsersTF.go

```diff
@@ -38,7 +38,7 @@ const (
 	userID              = "user_id"
 	username            = "username"
 	providerEnvVar      = "RANCHER2_PROVIDER_VERSION"
-	awsProviderEnvVar   = "AWS_PROVIDER_VERSION"
+	cloudProviderEnvVar   = "CLOUD_PROVIDER_VERSION"
```

Comment:
Can you make sure that the identing is the same as the above parameters? Just need to shift the `=` to the left I think two times?